pandas/core/indexes/base.py

```diff
@@ -4017,8 +4017,52 @@ def unique(self, level=None):
         result = super(Index, self).unique()
         return self._shallow_copy(result)
 
-    @Appender(base._shared_docs['drop_duplicates'] % _index_doc_kwargs)
     def drop_duplicates(self, keep='first'):
+        """
+        Return Index with duplicate values removed.
+
+        The drop_duplicates method can remove occurences or whole sets
+        of duplicated entries in a pandas.Index object.
+
+        Parameters
+        ----------
+        keep : {'first', 'last', ``False``}, default 'first'
+            - 'first' : Drop duplicates except for the first occurrence.
+            - 'last' : Drop duplicates except for the last occurrence.
+            - ``False`` : Drop all duplicates.
+
+        Returns
+        -------
+        deduplicated : Index
+
+        See Also
+        --------
+        pandas.Series.drop_duplicates : equivalent method on pandas.Series
+
+        Examples
+        --------
+        Generate an pandas.Index with duplicate values.
+
+        >>> idx = pd.Index(['lama', 'cow', 'lama', 'beetle', 'lama', 'hippo'])
+
+        With the 'keep' parameter, the selection behaviour of duplicated values
```

Comment:
Change first sentence to

    The `keep` parameter controls  which duplicate values are removed.

Reply:
sentence updated in 0d604ac